beast/examples/phat_small/regress_check.py

```diff
@@ -59,8 +60,8 @@ def hdf5diff(fname1, fname2):
                         else:
                             hd.add_missing_name(sname+'/'+cname)
             else:
-                for cname, cvalue in hdfa[sname].items():
-                    if cname not in hdfb[sname].keys():
+                for cname, cvalue in list(hdfa[sname].items()):
+                    if cname not in list(hdfb[sname].keys()):
```

Comment:
list is needed indeed here, but you should only make it once before the for-loop so that you do not instanciate it many times.

Reply:
Fixed.